### PR TITLE
name-added

### PR DESCRIPTION
**Pull request by theMillenniumFalcon**

### CODE REVIEW

**Review by CodingSpecies — APPROVED** [review]
LGTM, thank you @theMillenniumFalcon 

for some reason, the package-lock.json has changes, but this has been happening for many pull reuqests. 
Thank you.

File: public/data/theMillenniumFalcon.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
+{
+    "name": "Nishank Priydarshi",
+    "bio": "Full Stack developer exploring Devops",
+    "avatar": "https://github.com/theMillenniumFalcon.png",
+    "links": [
+      {
+        "name": "Follow me on GitHub",
+        "url": "http://github.com/theMillenniumFalcon",
```
Comment (Panquesito7) [inline public/data/theMillenniumFalcon.json:8]:
```suggestion
        "url": "https://github.com/theMillenniumFalcon",
```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

File: public/data/theMillenniumFalcon.json [hunk 2]
```diff
@@ -0,0 +1,17 @@
+{
+    "name": "Nishank Priydarshi",
+    "bio": "Full Stack developer exploring Devops",
```
Comment (Panquesito7) [inline public/data/theMillenniumFalcon.json:3]:
```suggestion
    "bio": "Full Stack developer exploring DevOps",
```

File: public/data/theMillenniumFalcon.json (hunk 1 above)
Comment (theMillenniumFalcon) [reply public/data/theMillenniumFalcon.json]: link changed

File: public/data/theMillenniumFalcon.json (hunk 2 above)
Comment (theMillenniumFalcon) [reply public/data/theMillenniumFalcon.json]: DevOps corrected

**Review by CodingSpecies — APPROVED** [review]

**Review by Panquesito7 — APPROVED** [review]
LGTM 🚀 Thank you for your contribution! 😄👍

**Pull request merged**